.github/workflows/ci.yaml

```diff
@@ -59,6 +59,10 @@ jobs:
           docker-compose exec -T web php --version
           docker-compose exec -T web php --info | grep xdebug.mode || true
 
+      - name: Collabora server capabilities
+        run: |
+          docker-compose exec -T web curl http://collabora.test:9980/hosting/capabilities | json_pp
```

Comment:
Let's add -s (silent) to not show the download progress. But this is a very very clever solution, good job.